test/integration/long/execution-profile-tests.js

```diff
@@ -0,0 +1,93 @@
+"use strict";
+var assert = require('assert');
+var util = require('util');
+
+var helper = require('../../test-helper.js');
+var Client = require('../../../lib/client.js');
+var types = require('../../../lib/types');
+var utils = require('../../../lib/utils.js');
+var loadBalancing = require('../../../lib/policies/load-balancing.js');
+var DCAwareRoundRobinPolicy = loadBalancing.DCAwareRoundRobinPolicy;
+var WhiteListPolicy = loadBalancing.WhiteListPolicy;
+var ExecutionProfile = require('../../../lib/execution-profile.js').ExecutionProfile;
+
+describe('ProfileManager', function() {
```

Comment:
On Windows CI env, 6-node cluster might be too much.
Do you think we can use a 4 nodes (`2:2`) and move it to "short" category?